src/BaGet/Program.cs

```diff
@@ -45,7 +45,11 @@ await provider
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>();
+                .UseStartup<Startup>()
+                .UseKestrel(options =>
+                {
+                    options.Limits.MaxRequestBodySize = null;

```

Comment:
Let's set `MaxRequestBodySize` to `262_144_000` (250 MB) to match nuget.org's upload limit (see [this documentation](https://docs.microsoft.com/en-us/nuget/policies/nuget-faq#managing-packages-on-nugetorg)).